content/02.main-text.md

```diff
@@ -1,8 +1,23 @@
 ## Introduction {.page_break_before}
 
-Openness in research benefits the researchers who practice it [@doi:10.7554/eLife.16800], their scientific peers, and the public.
-Improving the availability of code and data helps ensure the methods are transparent and the results can be reproduced  [@doi:10.1038/s41562-016-0021].
-Publishing in an open access journal increases the number of readers and reviewers [@doi:10.1371/journal.pone.0000308].
+The way science is performed is undergoing a terrific transition towards openness.
+The rise of the internet has enabled a global information commons, where scholarship can be publicly shared at a minimal cost.
+First, open access publishing provided an economic model that encourages maximal dissemination and reuse of scholarly articles [@doi:10.12688/f1000research.8460.3].
+Pressure from funders, libraries, self-archiving, and piracy are set to cement open access as the primary model for publishing the scientific record [@doi:10.7287/peerj.preprints.3119v1, @doi:10.7554/eLife.16800; @doi:10.7554/eLife.32822; @doi:10.1038/d41586-018-05754-1].
```

Comment:
Quick comment: mix of commas and semicolons separating references. This is unintentional and we should stick to semicolons, right?